db/db.go

```diff
@@ -220,6 +240,75 @@ func (db *DB) RevokeSSH(rci *RevokedCertificateInfo) error {
 	}
 }
 
+// GetRevokedCertificates gets a list of all revoked certificates.
+func (db *DB) GetRevokedCertificates() (*[]RevokedCertificateInfo, error) {
+	entries, err := db.List(revokedCertsTable)
+	if err != nil {
+		return nil, err
+	}
+	var revokedCerts []RevokedCertificateInfo
+	now := time.Now().Truncate(time.Second)
+
+	for _, e := range entries {
+		var data RevokedCertificateInfo
+		if err := json.Unmarshal(e.Value, &data); err != nil {
+			return nil, err
+		}
+
+		if !data.RevokedAt.IsZero() && data.RevokedAt.After(now) {
+			revokedCerts = append(revokedCerts, data)
```

Comment:
There won't be certificates revoked after now. While testing this I've only seen the certificate I've just revoked because this method is called just after the revocation, and with the truncation of now, this condition might be true.

Reply:
The CRL must contain all certificates that are still valid or will be valid but have been revoked.

Reply:
I removed the `After()` check, im unsure why it was there, i was probably confused about the date calculation? Let me know if its the right calc now, im not confident i havent done something fuzzy :)